### PR TITLE
add pco upgrade journey blog 26.4.2023

### PR DESCRIPTION
**Pull request by frosty-geek**

### CODE REVIEW

**Review by neuroserve — APPROVED** [review]
Very noice.

File: _i18n/en/_posts/blog/2023-04-06-scs-r4-pco-journey.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title: "Journey Through Two Years of pluscloud open: A Recap of the SCS Stack Upgrade Path"
```
Comment (maxwolfs) [inline _i18n/en/_posts/blog/2023-04-06-scs-r4-pco-journey.md:3]: Isn't Pluscloud Open a brand/produc name and should be in capital letters?

**Review by maxwolfs — APPROVED** [review]
LGTM!

File: _i18n/en/_posts/blog/2023-04-06-scs-r4-pco-journey.md (hunk 1 above)
Comment (fkr) [reply _i18n/en/_posts/blog/2023-04-06-scs-r4-pco-journey.md:3]: afaik the brandname is ‘pluscloud open’ (all lower caps)

Comment (maxwolfs) [reply _i18n/en/_posts/blog/2023-04-06-scs-r4-pco-journey.md:3]: Ah, okay thanks! I thought I have seen it on the website with capital letters...

**Comment by frosty-geek** [comment]
We will hold back on the blog post until pco prod regions are updated

/remind-me 2023-04-26 Publish this blog posting

File: _i18n/en/_posts/blog/2023-04-06-scs-r4-pco-journey.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+Throughout this journey, we have been grateful to be part of an open and welcoming community around the Sovereign Cloud Stack. We take pride in having contributed to and supported this community.
+
+{% asset 'blog/2023-scs-r4-pco-journey-sky-is-the-limit.jpg' vips:format='.webp' style="width:100%; max-height: 450px; object-fit: contain;" %}
+
+pluscloud open has two productive public environments, prod1 and prod2, located in Cologne and Hamburg, respectively. prod1 went live in July 2021 with the SCS pre-R0 release, while prod2 was launched in March 2023, both running now the latest release SCS R4. It's important to stress that this achievement was made possible by the SCS community, which has kept up with the supported OpenStack releases and enabled us to perform rolling upgrades without major downtime.
+
+
+As part of our commitment to building and improving a sustainable and data-sovereign Europe, we will continue to contribute to the Sovereign Cloud Stack Community. We aim to build a community of practice and transparency for operations using Open Operations, and we look forward to seeing what the future holds for the Sovereign Cloud Stack Community.
```
Comment (berendt) [inline _i18n/en/_posts/blog/2023-04-06-scs-r4-pco-journey.md:29]: I think it makes sense to add a link to https://openoperations.org here.

Comment (frosty-geek) [reply _i18n/en/_posts/blog/2023-04-06-scs-r4-pco-journey.md:29]: good point!

We will also include some more details about the recents weeks and the journey to 5.1.0 as well

**Comment by garloff** [comment]
Shouldn't we get this completed?
Based on the review comments, there is not much missing to finalize this, no?

**Review by maxwolfs — APPROVED** [review]
LGTM

**Pull request merged**